include/ghex/structured/rma_range_generator.hpp

```diff
@@ -11,6 +11,7 @@
 #ifndef INCLUDED_GHEX_STRUCTURED_RMA_RANGE_GENERATOR_HPP
 #define INCLUDED_GHEX_STRUCTURED_RMA_RANGE_GENERATOR_HPP
 
+//#include <thread>
```

Comment:
why is this commented out? can it be removed?

Reply:
fixed

Reply:
fixed